client/my-sites/email/email-providers-stacked-comparison/provider-cards/professional-email-card.tsx

```diff
@@ -0,0 +1,35 @@
+import { Gridicon } from '@automattic/components';
+import { translate } from 'i18n-calypso';
+import React from 'react';
+import poweredByTitanLogo from 'calypso/assets/images/email-providers/titan/powered-by-titan.svg';
+import { getTitanProductName } from 'calypso/lib/titan';
+import { ProviderCard } from 'calypso/my-sites/email/email-providers-stacked-comparison';
```

Comment:
Can we import this as a type?
```suggestion
import type { ProviderCard } from 'calypso/my-sites/email/email-providers-stacked-comparison';
```